docs/typescript.md

```diff
@@ -23,23 +22,24 @@ start.
 
 ## Setup
 
-We're assuming you're using ES6 for both browser (for example with React), or nodejs.
-
-Install [Yarn](https://yarnpkg.com/) or [Node.js](https://nodejs.org/en). Init a new project with
-`npm init`. Install Noir dependencies in your project by running:
+We're assuming you're using ES6 for both browser (for example with React), or nodejs. Install [Node.js](https://nodejs.org/en). Init a new project with `npm init`. Install Noir dependencies in your project by running:
 
 ```bash
-npm i @noir-lang/noir_wasm@0.3.2-fa0e9cff github:noir-lang/barretenberg#39a1547875f941ef6640217a42d8f34972425c97 @noir-lang/aztec_backend@0.1.0-0c3b2f2
+npm i @aztec/bb.js github:noir-lang/acvm-simulator-wasm.git#c56eec56f67f90fef90126c5575b85190bdcd1e1 github:noir-lang/noir_wasm.git fflate ethers@5.7.2
```

Comment:
Since we are using `Buffer`, VS Code is suggesting I also `npm i --save-dev @types/node`